pkgs/by-name/on/one-click-backup/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, qt6
+#, wrapQtAppsHook
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "one-click-backup";
+  version = "1.2.2.1";
```

Comment:
This version doesn't exist on the remote. The latest pushed tag is [1.2.2](https://gitlab.com/dev-nis/nis-one-click-backup-qt/-/tree/1.2.2?ref_type=tags).

```
error: builder for '/nix/store/c4v512z51l29bh34avhq53hbbjgh4ak3-source.drv' failed with exit code 1;
       last 7 log lines:
       >
       > trying https://gitlab.com/api/v4/projects/dev-nis%2Fnis-one-click-backup-qt/repository/archive.tar.gz?sha=1.2.2.1
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0    32    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix-store -l /nix/store/c4v512z51l29bh34avhq53hbbjgh4ak3-source.drv'.
error: 1 dependencies of derivation '/nix/store/s3glb7bvls4n7snil7mjayl5wyax75gk-one-click-backup-1.2.2.1.drv' failed to build
```

Reply:
Oh, I somehow forgot to publish a new release (I only added a new tag to git). Should be fixed now: [Version 1.2.2.1](https://gitlab.com/dev-nis/nis-one-click-backup-qt/-/releases/1.2.2.1)